.travis.yml

```diff
@@ -1,3 +1,10 @@
 language: java
+
 sudo: false
 
+jdk:
+ - oraclejdk7
```

Comment:
You may want to change this to `openjdk6` if you intend to support Java 6, I was not sure.